lib/content/audio/predictions.ex

```diff
@@ -74,8 +83,9 @@ defmodule Content.Audio.Predictions do
           end
 
         {prefix, suffix} = if(platform_prefix?, do: {qualifier, []}, else: {[], qualifier})
+        four_cars = if four_cars?(audio), do: [:four_car_train_message], else: []
 
-        [the_next_or_following] ++ train ++ prefix ++ status ++ suffix ++ followup
+        [the_next_or_following] ++ train ++ prefix ++ status ++ suffix ++ followup ++ four_cars
```

Comment:
`followup` here is possibly something like "we will announce the platform for boarding soon", so does it make sense to then say "move to the front of the platform to board", when riders don't yet know _which_ platform they should move to the front of?

Reply:
Fortunately (?) that followup message will only trigger on the JFK mezzanine, where we don't do passive readouts for four-car predictions. So as of now, this won't be an issue, but this is a good callout because there's a lot of stuff piling up here, and we could easily back ourselves into some nonsensical messaging if we're not careful about it.

Reply:
> that followup message will only trigger on the JFK mezzanine, where we don't do passive readouts for four-car predictions

Since I didn't see JFK-specific logic in these changes, I'm guessing this is because of a general property of mezzanine signs, but I'm not quite connecting the dots.